### PR TITLE
UI enhancements

### PR DESCRIPTION
**Pull request by luisgov**
* Make manual TO link more obvious
story: https://www.pivotaltracker.com/n/projects/2160940/stories/160397755
screen shot 2018-09-13 at 14 31 30

---

* Add placeholder text suggesting environment names
story: https://www.pivotaltracker.com/n/projects/2160940/stories/160396394
screen shot 2018-09-13 at 09 19 07

---

*  Improve request form headers clarity
story: https://www.pivotaltracker.com/story/show/160397912
screen shot 2018-09-13 at 09 30 22

screen shot 2018-09-13 at 09 34 06

---

* Update annual spend +1M message
story: https://www.pivotaltracker.com/story/show/160398070
screen shot 2018-09-13 at 09 40 43

---

* Add help text for "Total Spend"
story: https://www.pivotaltracker.com/n/projects/2160940/stories/160398315
screen shot 2018-09-13 at 09 49 17

---
* Fix grammar error on Service Branch question
story: https://www.pivotaltracker.com/story/show/160398466

screen shot 2018-09-13 at 10 15 13

---
* Update Copy for Workspace and CSP Roles
story: https://www.pivotaltracker.com/n/projects/2160940/stories/160298069
screen shot 2018-09-13 at 11 04 59

screen shot 2018-09-13 at 11 22 47

---

* Change all instance of "JEDI" to "JEDI Cloud"
story: https://www.pivotaltracker.com/story/show/159408035

### CODE REVIEW

**Comment by andrewcroce** [comment]
I should have communicated this addition earlier, but the Alert has the ability to display Icon Links where you have the button. The blue button you have is definitely more visible, but looks a little garish. Maybe we try a _touch_ more subtle before doing this?

You can do:

```
{{ Alert('Some Alert',
    message="<p>Some Message...</p>",
    level='warning',
    actions=[
        {
            "label": "Some Action",
            "href": "/some/action/url",
            "icon": "some-icon"
        }
    ]
  ) }}
```

File: templates/requests/_new.html [hunk 1]
```diff
@@ -17,8 +17,8 @@
     <div class="panel">
 
       <div class="panel__heading">
-        <h1>New Request</h1>
-        <div class="subtitle">{% block subtitle %}{% endblock %}</div>
+        <h1>{% block heading %}{% endblock %}</h1>
+        <div class="subtitle"><h2>New Request</h2></div>
```
Comment (andrewcroce) [inline templates/requests/_new.html]: There shouldn't be an `h2` tag here. A subtitle isn't the same as a sub heading, ie it does not denote a document sub section of the `h1`

File: templates/requests/financial_verification.html [hunk 2]
```diff
@@ -20,8 +20,8 @@
     {% set extended_url = url_for('requests.financial_verification', request_id=request_id, extended=True) %}
     {{ Alert('Task Order not found in EDA',
         message="We could not find your Task Order in our system of record, EDA.
-                 Please confirm that you have entered it correctly.
-                 <a href=\"%s\">Otherwise enter TO information manually.</a>
+                 Please confirm that you have entered it correctly.<br>
+                 <a class=\"usa-button usa-button-secondary\" href=\"%s\">Enter Task Order information manually</a>
```
Comment (andrewcroce) [inline templates/requests/financial_verification.html]: See my comment about the icon links

File: templates/requests/_new.html (hunk 1 above)
Comment (luisgov) [reply templates/requests/_new.html]: 👍

File: templates/requests/financial_verification.html (hunk 2 above)
Comment (luisgov) [reply templates/requests/financial_verification.html]: fixed

**Pull request merged**